plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/StaticAnalyzerPreferences.java

```diff
@@ -140,7 +140,7 @@ private void createAdvancedContents(Composite parent) {
 		final Composite callGraphContainer = new Composite(staticAnalysisGroup, SWT.None);
 		callGraphContainer.setLayout(new GridLayout(2, true));
 		final Label label1 = new Label(callGraphContainer, SWT.SHADOW_IN);
-		label1.setText("Call-graph construction algorithm");
+		label1.setText("Call-graph constructiomarkern algorithm");

```

Comment:
Is that a typo?

Reply:
yes it is, I will fix it, thanks for noticing.